app/src/main/java/com/sdpteam/connectout/event/EventBuilder.java

```diff
@@ -0,0 +1,62 @@
+package com.sdpteam.connectout.event;
+
+import com.google.android.gms.maps.model.LatLng;
+
+public class EventBuilder {
+
+    public EventBuilder() {
+    }
+
+    public EventBuilder(LatLng latLng) {
```

Comment:
Yes, but using creating a new class in the perspective of issue #19 is useless.
Moreover, by doing so, we lose all experience that provides `LatLng` for positioning.
This might be more relevant for issues #20 #21.